__tests__/__integration__/cmd/src/cli/validate/string-or-empty/string-or-empty.definition.ts

```diff
@@ -0,0 +1,35 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { ICommandDefinition } from "../../../../../../../packages/index";
+
+// Range for option length-range
+const LENGTH_RANGE_START: number = 2;
+const LENGTH_RANGE_END: number = 8;
+
+// Range for option eggs
+const EGGS_RANGE_START: number = 1;
+const EGGS_RANGE_END: number = 12;
+
+export const stringOrEmptyTestCommand: ICommandDefinition = {
+    name: "string-or-empty",
+    description: "Test string or empty option.",
+    type: "command",
+    handler: __dirname + "/string-or-empty.handler",
+    options:
+        [
+            {
+                name: "string-or-empty",
```

Comment:
When I issue, `ts-node ./ validate string-or-empty`, from the cli directory it looks like validation passes and I assume that the value of `string-or-empty` is undefined? Or is the value ""?

If the value is "", how would this affect the case of specifying a default value and then omitting the option from the command. Is the default used instead of ""?

Perhaps you should echo the output of the option in your validation command to help understand this concept.

Reply:
Also, what happens if this option is required. I would expect that you now have to specify `--string-or-empty ""`, is this assumption correct?